engine/crates/composition/src/composition_ir.rs

```diff
@@ -202,7 +138,8 @@ pub(crate) struct FieldIr {
     pub(crate) parent_name: subgraphs::StringId,
     pub(crate) field_name: subgraphs::StringId,
     pub(crate) field_type: subgraphs::FieldTypeId,
-    pub(crate) arguments: Vec<(subgraphs::StringId, subgraphs::FieldTypeId)>,
+    /// (argument_name, argument_type, composed_directives)
```

Comment:
Worth a struct if you need a comment :)

Reply:
True :/ I'm lazy about creating structs, but there's no excuse here.

Reply:
Fixed and pushed.